engine/src/main/java/org/terasology/engine/rendering/logic/FloatingTextRenderer.java

```diff
@@ -77,7 +77,7 @@ private void render(Iterable<EntityRef> floatingTextEntities) {
             LocationComponent location = entity.getComponent(LocationComponent.class);
 
             if (location == null) {
-                logger.warn("location component is not defined can't render text: {}", floatingText.text);
+                logger.atWarn().addArgument(() -> floatingText.text).log("location component is not defined can't render text: {}");
```

Comment:
Simple member access
```suggestion
                logger.warn("location component is not defined can't render text: {}", floatingText.text); //NOPMD
```